examples/chip-tool/commands/common/CHIPCommand.h

```diff
@@ -222,11 +224,16 @@ class CHIPCommand : public Command
     chip::Optional<char *> mCDTrustStorePath;
     chip::Optional<bool> mUseMaxSizedCerts;
     chip::Optional<bool> mOnlyAllowTrustedCdKeys;
+    chip::Optional<char *> mDacRevocationSetPath;
 
     // Cached trust store so commands other than the original startup command
     // can spin up commissioners as needed.
     static const chip::Credentials::AttestationTrustStore * sTrustStore;
 
+    // Cached DAC revocation delegate, this can be set using "--dac-revocation-set-path" argument
+    // Once set this will be used by all commands.
```

Comment:
This information should be in the AddArgument help message, no?  People should not have to read the source to find out how arguments behave.

Reply:
Added it to the argument help!